### ISSUE TITLE
Latest data fetch URL return nothing

### ISSUE THREAD

**Issue by kabirnayeem99** [aminul-palash/islamqaAPI#1]
127.0.0.1:8000/api/latest or "GET /api/latest HTTP/1.1" returns just this text 'Latest Data Fetched from Islamic QA'.

**Comment by aminul-palash** [comment]
yes, this will only return 'Latest Data Fetched from Islamic QA'. and saved ! , you can see results in here "http://127.0.0.1:8000/api/questions/"

**Comment by aminul-palash** [comment]
please look: "API/views.py" file.
